src/dmclock_server.h

```diff
@@ -1292,6 +1340,18 @@ namespace crimson {
       } // pull_request
 
 
+      void update_client_info(const C& client_id) {
```

Comment:
I removed repeated update_client_info(s) in both PullPriorityQueue and PushPriorityQueue.